Assets/Scripts/Actors/ActorComponents/ActorCamera.cs

```diff
@@ -90,10 +90,10 @@ void CameraControl()
 
 			cam.transform.RotateAround(transform.position, 
 			                           cam.transform.right, 
-			                           Input.GetAxis("Mouse Y" + WadeUtils.platformName) * rotSpeed.y);
+			                           Input.GetAxis("Mouse Y" + PlatformUtils.platformName) * rotSpeed.y);
 			cam.transform.RotateAround(transform.position, 
 			                           cam.transform.up, 
-			                           Input.GetAxis("Mouse X" + WadeUtils.platformName) * rotSpeed.x);
+			                           Input.GetAxis("Mouse X" + PlatformUtils.platformName) * rotSpeed.x);
```

Comment:
Make InputUtils.GetPlatformAxis(string name). Shouldn't be too hard and will wrap up this ugly repeated code.